bin/workflows/pycbc_make_faithsim_workflow

```diff
@@ -0,0 +1,187 @@
+#! /usr/bin/env python
+
+"""
+Program for running a faithfulness comparisons workflow analysis between two approximants
+and generate files containing the match between them and plots.
+"""
+
+import os
+import pycbc
+import pycbc.workflow as wf
+import logging
+import configparser as ConfigParser
+import numpy as np
+import logging
+import argparse
+from pycbc.workflow.jobsetup import (
+    select_generic_executable,
```

Comment:
You don't use `select_generic_executable` so this line can be removed.

Reply:
I can't see any of the other things used here either? Maybe a generic point then to check for, and remove, unused imports.